src/containers/SubscriptionsPage/reducers.js

```diff
@@ -20,6 +20,6 @@ export const subscriptionsReducer = (state = fromJS({}), action = {}) => {
     case FETCH_SUBSCRIPTIONS_FAILURE:
       return state.merge({ subscriptions: [] });
     default:
-      return state;
+      return state.merge({ subscriptions: [] });
```

Comment:
Yes (and no). 

Initial state should be set (like you mentioned) in default params. We already have an `initialState` immutable object which we should reuse when possible, otherwise we'd have to rewrite the entire shape of our subscriptions state object: `fromJS({subscriptions: []})` is missing `loading` for example. Best to have that in just one place. 
```js
export const subscriptionsReducer = (state = initialState, action = {}) => { ... }
```

The second thing to note is that the `default` case should not modify state. State is always a function of current state and an action. When we don't recognise an action (e.g. if the action is `PAYMENT_METHODS_FETCHED`), we don't necessarily want to do anything with state. Our reducer should handle all actions that would affect our state, and in the default action (i.e. any other action) we just return state.